core/extraction/towns_fund_round_one.py

```diff
@@ -430,6 +430,39 @@ def transform_project_funding_profiles(
     merged_df = merged_df[merged_df["Project ID"] != "TD-20"]
     merged_df.loc[merged_df["Secured"] == "None", "Secured"] = ""
 
+    # drop always unused cells for funding secured before 2020 and beyond 2026
+    unused_mask = merged_df.loc[
+        (
+            (merged_df["Funding Source Type"] == "Towns Fund")
+            & (merged_df["Start_Date"].isna() | (merged_df["End_Date"].isna()))
+        )
+    ]
+    merged_df.drop(unused_mask.index, inplace=True)
+
+    unused_fhsf_mask = merged_df.loc[
+        # drop unused FHSF Questions
```

Comment:
can you just rename this bit in the comment, as it's potentially confusing  - its actually dropping unused FHSF Funding Sources (Funding Questions is a separate table)